AnkiDroid/src/main/java/com/ichi2/anki/dialogs/ImportDialog.kt

```diff
@@ -16,8 +16,8 @@
 
 package com.ichi2.anki.dialogs
 
+import android.app.AlertDialog
```

Comment:
```suggestion
import androidx.appcompat.app.AlertDialog
```
I wonder if both dialogs work the same way, but it's better to import the appcompat version here because our helper methods use it

Reply:
As much as I know `androidx.appcompat.app.AlertDialog` is based on `android.app.AlertDialog` but has additional features like backward compatibility, support for vector drawables, extended theming options, etc.

Reply:
The `androidx.appcompat.app.AlertDialog` should be used as all changed dialog use it(we have some uses of the platform AlertDialog but those are mostly older code) and we should strive for consistency. The app compat version is also the one used in our helpers from ALertDialogFacade and it would allow to simply write `.negativeButton(string)` instead of `.setNegativeButton(string) { _,_ ->}`.

Reply:
Done ✅, now using `androidx.appcompat.app.AlertDialog`